guides/common/modules/con_foreman-on-aws-prerequisites.adoc

```diff
@@ -0,0 +1,4 @@
+[[Prerequisites]]
+= Prerequisites
+
+Before you can install and register {ProjectName} and {SmartProxy}, you must set up accounts with Amazon Web Services (AWS) and create and start {RHEL} instances on AWS.
```

Comment:
Personally, I think this sounds too obvious to mention as a prerequisite. It's your call.

Reply:
Yes, I feel the same. -> Restructured and dropped this sentence in https://github.com/theforeman/foreman-documentation/pull/3485